### PR TITLE
Rollout improved example converter

### PR DESCRIPTION
**Pull request by t0yv0**
Rolls out improved TF example converter.

### CODE REVIEW

**Comment by VenelinMartinov** [comment]
`RandomInteger.java` has

`.action(%!v(PANIC=Format method: runtime error: invalid memory address or nil pointer dereference))`

Are we printing a go panic in the example?

**Comment by t0yv0** [comment]
Yeah let me have a closer look.

File: sdk/java/src/main/java/com/pulumi/random/RandomInteger.java [hunk 1]
```diff
@@ -53,19 +52,15 @@
...
  * 
- *         var main = new ListenerRule(&#34;main&#34;, ListenerRuleArgs.builder()        
+ *         var main = new AlbListenerRule(&#34;main&#34;, AlbListenerRuleArgs.builder()        
  *             .listenerArn(priority.keepers().applyValue(keepers -&gt; keepers.listenerArn()))
  *             .priority(priority.result())
- *             .actions(ListenerRuleActionArgs.builder()
- *                 .type(&#34;forward&#34;)
- *                 .targetGroupArn(var_.target_group_arn())
- *                 .build())
+ *             .action(%!v(PANIC=Format method: runtime error: invalid memory address or nil pointer dereference))
```
Comment (t0yv0) [inline sdk/java/src/main/java/com/pulumi/random/RandomInteger.java:61]: Yes this is a regression I would block on.

**Comment by t0yv0** [comment]
Thanks for spotting @VenelinMartinov blocked on https://github.com/pulumi/pulumi-java/issues/1341

**Comment by t0yv0** [comment]
Resolved panics by upgrading AWS pinned reference to 6.x.

**Review by VenelinMartinov — APPROVED** [review]
🔥

**Pull request merged**